x/denommetadata/keeper/msg_server.go

```diff
@@ -22,6 +21,7 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 
 var _ types.MsgServer = msgServer{}
 
+// Deprecated
```

Comment:
deprecated means its use is discouraged, and it will be removed in the future, so we avoid rug pull.

Anyway, the current decision is to go forth with the rug pull

Reply:
what do you mean by rug pull here?

Reply:
I mean, say you have an API endpoint that is being used, and instead of gently letting everyone know that this endpoint will be removed in the future at some point. But if you just straight up remove it, it's kind of like a rug pull, innit?

But, regardless, this is going to be removed now because it's safer that way